src/rules/block-closing-brace-newline-after/index.js

```diff
@@ -7,12 +7,12 @@ import {
 export const ruleName = "block-closing-brace-newline-after"
 
 export const messages = ruleMessages(ruleName, {
-  expectedAfter: () => "Expected newline after \"}\"",
-  rejectedAfter: () => "Unexpected space after \"}\"",
-  expectedAfterSingleLine: () => "Expected single space after \"}\" of a single-line block",
-  rejectedAfterSingleLine: () => "Unexpected space after \"}\" of a single-line block",
-  expectedAfterMultiLine: () => "Expected single space after \"}\" of a multi-line block",
-  rejectedAfterMultiLine: () => "Unexpected space after \"}\" of a multi-line block",
+  expectedAfter: () => `Expected newline after "}"`,
+  rejectedAfter: () => `Unexpected space after "}"`,
+  expectedAfterSingleLine: () => `Expected single space after "}" of a single-line block`,
+  rejectedAfterSingleLine: () => `Unexpected space after "}" of a single-line block`,
+  expectedAfterMultiLine: () => `Expected single space after "}" of a multi-line block`,
+  rejectedAfterMultiLine: () => `Unexpected space after "}" of a multi-line block`,
```

Comment:
In the future, it would make the PR easier to read and understand if strictly cosmetic changes like this are saved for a separate cosmetic PR. But I get it, we're committing fast and furious.